lib/constable_web/views/announcement_view.ex

```diff
@@ -3,35 +3,33 @@ defmodule ConstableWeb.AnnouncementView do
 
   def json_interests(interests) do
     interests
-    |> Enum.map(&(%{name: &1.name}))
-    |> Poison.encode!
+    |> Enum.map(&%{name: &1.name})
+    |> Poison.encode!()
   end
 
   def comma_separated_interest_names(interests) when is_list(interests) do
     interests
-    |> Enum.map(&(&1.name))
+    |> Enum.map(& &1.name)
```

Comment:
I'm assuming the parens changes are auto-formatting as well?

Reply:
Yeap. I've turned off my auto-formatter in the past. I think I might just open a PR soon that auto-formats everything, and then this won't be a problem.